_sass/hero.scss

```diff
@@ -42,10 +43,9 @@
 
 	a {
```

Comment:
this seems to be another global css property

Reply:
hey h1, a tags are under the hero section.